app/containers/App/index.js

```diff
@@ -76,27 +71,17 @@ export class App extends Component {
         {/* eslint-disable-next-line */}
         <main className={`container ${menuOpen ? 'desktop-only' : ''}`} onClick={() => this.onBackgroundClick()}>
           {shouldShowSpinner && <Spinner />}
-          {!shouldShowTerms && (
-            <div className="main-content">
-              <Switch>
-                {routes.map(route => (
-                  <RouteWithSubRoutes key={route.name} {...route} />
-                ))}
-              </Switch>
-            </div>
-          )}
-          {shouldShowTerms && <Terms close={() => hideTermsAndConditions()} />}
+          <div className="main-content">
+            <Switch>
+              {routes.map(route => (
+                <RouteWithSubRoutes key={route.name} {...route} />
+              ))}
+            </Switch>
+          </div>
         </main>
         {/* eslint-disable-next-line */}
         <div onClick={() => this.onBackgroundClick()}>
-          <Footer
-            meta={{
-              items: [
-                { text: 'Contact us', href: `mailto:${mailTo}` },
-                { text: 'Terms and conditions', ...linkOnClick(showTermsAndConditions) },
-              ],
-            }}
-          />
+          <FooterContainer feedbackEmail={mailTo} prisonStaffHubUrl={prisonStaffHubUrl} />
```

Comment:
Do you think it's worth testing that the correct mailTo and prisonStaffHubUrl are being passed to this component? It would also increase the test coverage on this project.